{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/config.py

```diff
@@ -115,14 +115,17 @@ def _(x):
 # http://flask.pocoo.org/docs/0.12/config/#builtin-configuration-values
 
 #: Secret key - each installation (dev, production, ...) needs a separate key.
+# It should be changed before deploying.
```

Comment:
What happens if you just completely remove it?

Reply:
Also, see comment on the single-line sphinx docstring (``#:``)

Reply:
It is set to CHANGE_ME by default, shall we remove it from here then and we add it to a security/deployment section in the main documentation?